src/app/containers/Caption/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+import Caption from '../../components/Figure/Caption';
+import Text from '../../components/Text';
+import { ServiceContext } from '../../components/ServiceContext';
+import VisuallyHiddenText from '../../components/VisuallyHiddenText';
+
+const CaptionWithOffscreenText = ({ children }) => (
+  <ServiceContext.Consumer>
+    {({ imageCaptionOffscreenText }) => (
+      <Caption>
+        <VisuallyHiddenText>{imageCaptionOffscreenText}</VisuallyHiddenText>
```

Comment:
We should conditionally render `<VisuallyHiddenText>`, so if `imageCaptionOffscreenText` is undefined, we don't render and empty span here.  We should definitely have a unit test to capture this usecase.

Reply:
Done, & as discussed in person, we've added `imageCaptionOffscreenText` properties to the News & Persian service configs.